file-avail-s3.md

```diff
@@ -46,18 +46,31 @@ methods, HTCondor file transfer or SQUID, and when these files are
 expected to be accessed outside of CHTC. This includes individual input files
 greater than 100MB and individual output files greater than 3-4GB.
 
+Files in our S3 data storage are organized in storage units called
+"buckets." You can think of an S3 bucket like a folder containing a
+set of data. Each bucket has a unique name of your choosing and can
+contain folders, executable files, data files, and most other types of
+files. S3 buckets are protected with a key that is unique to you
+(similar to a password) and, when provided with the key, buckets
+can be accessed from any machine with an internet connection. CHTC
+automatically creates and manages keys for users, so you do not have
+to provide your key when manging files in your S3 buckets on CHTC
+transfer servers or when submitting jobs on CHTC submit servers that
+transfer data from S3 buckets.
+
```

Comment:
Is this the right place for this?

Reply:
I think so? Otherwise, you don't explain what buckets are until you've already used the term several times.